pkg/agent/controller/agent.go

```diff
@@ -167,20 +219,40 @@ func (a *Controller) serviceExportToRemoteServiceImport(obj runtime.Object, op s
 	}
 
 	serviceImport.Spec = lighthousev2a1.ServiceImportSpec{
-		Type: lighthousev2a1.SuperclusterIP,
+		Type: svcType,
 	}
+
 	serviceImport.Status = lighthousev2a1.ServiceImportStatus{
 		Clusters: []lighthousev2a1.ClusterStatus{
-			a.newClusterStatus(svc, a.clusterID),
+			{
+				Cluster: a.clusterID,
+				IPs:     a.getIpsForService(svc, svcType),
+			},
 		},
 	}
 
+	if svcType == lighthousev2a1.SuperclusterIP {
+		serviceImport.Annotations[clusterIP] = serviceImport.Status.Clusters[0].IPs[0]
```

Comment:
Suggested comment: "_We also store the clusterIP in an annotation as an optimization to recover it in case the IPs are cleared out when here's no backing Endpoint pods_."

Reply:
Done